src/_pytest/unittest.py

```diff
@@ -218,11 +218,12 @@ def setup(self) -> None:
         super().setup()
 
     def teardown(self) -> None:
-        super().teardown()
```

Comment:
This change is really related and doesn't have any effect because the super call doesn't do anything, but nonetheless teardown must be done in reverse order, so I fixed it just in case the super ever starts doing something.